tests/device_tests/bitcoin/test_signtx.py

```diff
@@ -322,12 +318,12 @@ def test_one_three_fee(client):
                 request_output(1),
                 messages.ButtonRequest(code=B.ConfirmOutput),
                 request_output(2),
+                messages.ButtonRequest(code=B.ConfirmOutput),
```

Comment:
This doesn't look fixable with this data: all three outputs go out to another account. Here we really want two normal and one change output.

Reply:
Addressed in eaaa13742bdbe50f06a8452d8f6a93a52365a211 - created a new transaction to comply with this requirement